articles/virtual-desktop/whats-new.md

```diff
@@ -69,6 +69,8 @@ To learn more, see [our blog post](https://azure.microsoft.com/updates/windows-v
 
 ## Next steps
 
+Keep up to date with future capabilites and updates by reviewing the Microsoft 365 [Windows Virtual Desktop roadmap](https://www.microsoft.com/en-us/microsoft-365/roadmap?filters=Windows%20Virtual%20Desktop)
```

Comment:
Learn about future plans at the [Microsoft 365 Windows Virtual Desktop roadmap](https://www.microsoft.com/microsoft-365/roadmap?filters=Windows%20Virtual%20Desktop).
```suggestion
Keep up to date with future capabilites and updates by reviewing the Microsoft 365 [Windows Virtual Desktop roadmap](https://www.microsoft.com/en-us/microsoft-365/roadmap?filters=Windows%20Virtual%20Desktop)
```

Reply:
How about?
Learn about future plans and features at the [Microsoft 365 Windows Virtual Desktop roadmap](https://www.microsoft.com/microsoft-365/roadmap?filters=Windows%20Virtual%20Desktop).